jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -1129,7 +1128,7 @@ def _load_regions(self, regions, combination_mode=None, max_num_regions=None,
                 else:
                     combo_mode = combination_mode
 
-                if combo_mode == 'new':
+                if combo_mode == 'new' or combo_mode is None:
```

Comment:
how do we best notify users of this behavior change?  Just loudly in the changelog and/or docs?  Warning message when `combo_mode is None`?

Reply:
I think changelog and docs should be enough but I can definitely make it louder if people think that would be helpful.